### PR TITLE
Wizard Fireball Auto-Disable

### PR DESCRIPTION
**Pull request by GunHog**
:cl: Gun Hog
tweak: Wizard (and Devil) fireballs now automatically toggle off once fired.
/:cl:

As requested in https://tgstation13.org/phpBB/viewtopic.php?f=10&t=8508

### CODE REVIEW

File: code/modules/spells/spell_types/wizard.dm [hunk 1]
```diff
@@ -300,6 +300,7 @@
 
 	var/list/targets = list(target)
 	perform(targets,user = ranged_ability_user)
+	remove_ranged_ability() //Auto-disable the ability once performed
```
Comment (ChangelingRain) [inline code/modules/spells/spell_types/wizard.dm]: this should only happen if the spell successfully casts

**Comment by GunHog** [comment]
Moved it down there since it is after all the magic code!

**Comment by ExcessiveUseOfCobblestone** [comment]
JUST MOVE THE BUTTON WHEN YOU'RE USING FIREBALL REEEEE

**Comment by A-t48** [comment]
Thank you. Hit this issue playing Devil the other day.

**Pull request merged**